packages/dev/core/src/Meshes/mesh.ts

```diff
@@ -445,7 +445,21 @@ export class Mesh extends AbstractMesh implements IGetSetVerticesData {
     /**
      * Use this property to change the original side orientation defined at construction time
      */
-    public overrideMaterialSideOrientation: Nullable<number> = null;
+    public sideOrientation: Nullable<number> = null;
+
+    /**
+     * @deprecated Please use materialSideOrientation instead
+     */
+    public get overrideMaterialSideOrientation(): Nullable<number> {
+        return this.sideOrientation;
+    }
+
+    /**
+     * @deprecated Please use materialSideOrientation instead
+     */
+    public set overrideMaterialSideOrientation(value: Nullable<number>) {
+        this.sideOrientation = value;
+    }

```

Comment:
I'm not sure this is right. If we must keep the backwards compatibility for this property, then it needs to be its own property and handled the same way as before because the behavior is now different. This sounds complicated. I'm not sure we should keep backwards compatibility for this property. Can we check if anyone uses this property besides glTF loader?

Reply:
@bghgary checking the code, it is being used on Mesh, EffectLayer, DepthRenderer, DecalBuilder, GeometryBufferRenderer and ShadowGenerator, besides the GLTF Loader and Exporter: https://github.com/search?q=repo%3ABabylonJS%2FBabylon.js+overridematerialsideorientation&type=code